generic3g/specs/FieldSpec.F90

```diff
@@ -201,7 +201,7 @@ function new_FieldSpec_geom(unusable, geom, vertical_grid, vertical_dim_spec, ty
       ! regrid_param
 
       if (present(default_value)) field_spec%default_value = default_value
-      field_spec%accumulation_type = NO_ACCUMULATION
+      field_spec%accumulation_type = INSTANTANEOUS
```

Comment:
This should go directly into the FrequencyAspect.  No need to separately store it.

Reply:
Done